src/axolotl/utils/data.py

```diff
@@ -60,10 +60,12 @@ def load_tokenized_prepared_datasets(
         else Path(default_dataset_prepared_path) / ds_hash
     )
     dataset = None
+    use_auth_token = False
```

Comment:
Can we just keep it to `True`? I do not see the benefit to setting it to `False`.

Reply:
if it's true, the user needs to log into huggingface-cli before running

Reply:
This would mean. If it's a private repo, the auth token will not be included for Line 68?

Reply:
good call, i'll revert this part